src/test/resources/data/StudentHomePageUiTest.json

```diff
@@ -190,6 +190,7 @@
   },
   "students": {
     "alice.tmms@SHomeUiT.CS2104": {
+      "googleId": "",
```

Comment:
Just making a note that we should address the inconsistencies in `googleId` for other `.json` files too. Not that important for now I suppose.

Reply:
I don't think we should add an empty string here.

In doing this, we are saying the googleID of the student is an empty string but not null. This breaks the assumption that students who don't register should not have googleID attribute (may cause "bug" in the future even it is test data). There are differences between null and empty string in the datastore.

Reply:
I agree with you but due to our broken design of `AccountAttributes` in the first place, we are stuck with this until we change it appropriately.

In fact, an empty string and null are conflated in several locations throughout the TEAMMATES codebase.

The unfortunate thing is we can't write comments in JSON files but an empty string is a more accurate representation of how our codebase initializes students at the current point.

Do you have any suggestions or anything you are particularly against because of this empty string because I think it's rather more accurate at this point (except being inconsistent with all the other test data).

Reply:
It's OK. We can leave it there. We already take care of both cases.
https://github.com/TEAMMATES/teammates/blob/c5e80a997c8848977d77d9c695b8751597090739/src/main/java/teammates/common/datatransfer/attributes/StudentAttributes.java#L101-L103